tests/unity/wrap_test/src/call/call.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <call.h>
```

Comment:
```suggestion
#include "call.h"
```

Reply:
that depends on whether this is actually a test of `"`'s handled correctly.

Reply:
I dont think it is. If it is, then something should fail on changing this.

Reply:
Not necessarily.

I assume code can handle both `#include "something"` and `#include <something>`.
Code will only fail if it cannot handle this, but with changing this you'll never know if it can handle it.

Reply:
@balaji-nordic Ref: https://github.com/nrfconnect/sdk-nrf/pull/2182

Reply:
@gregersrygg  Seems you have already changed this. Could you change it back if you find out that this is actually a test?

Reply:
@tejlmand The wrapper scripts only parses the header files, so I think it's safe to change it here.